examples/train-allcnn-cifar.sh

```diff
@@ -5,11 +5,12 @@
 #
 # CIFAR-10
 #
-# All-CNN 
+# All-CNN
 #
 # no-reg bn heavier no-inv
+
 python train.py \
---data_file ~/datasets/hdf5/cifar10.hdf5 \
+--data_file ~/Storage/Datasets/cifar10/hdf5/cifar10.hdf5 \
```

Comment:
I'd leave the original paths in the public repository, which are meant to be generic. These scripts should serve as "examples", not as the go-to executable files.